sw/device/lib/base/freestanding/stdbool.h

```diff
@@ -12,11 +12,15 @@
  * This header implements the stdbool.h standard header, as required by C11
  * S4p6. This header is specified in detail in S7.18 of the same.
  */
+#ifndef __cplusplus
+#define _Bool int
```

Comment:
why int instead of char? @beeeeep

Reply:
No reason, but char is better. Changing it now